### PR TITLE
Iterating on Slack Onboarder

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@andrewconner @JRuff42

### CODE REVIEW

**Comment by ryanpbrewster** [comment]
Don't merge this until the "please authenticate through slack" flow works

File: kifi-backend/modules/common/app/com/keepit/common/Implicits.scala [hunk 1]
```diff
@@ -94,6 +94,13 @@ final class EitherExtensionOps[A, B](xs: Traversable[Either[A, B]]) {
   }
 }
 
+final class OptionExtensionOpts[A](x: Option[A]) {
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/common/Implicits.scala:97]: Why not use the built-in `.contains`? Does the same thing.

**Comment by andrewconner** [comment]
Can this go without the "authenticate with slack" flow? We won't that live just yet, but will want to QA this.

File: kifi-backend/modules/common/app/com/keepit/common/Implicits.scala (hunk 1 above)
Comment (ryanpbrewster) [inline kifi-backend/modules/common/app/com/keepit/common/Implicits.scala:97]: `aOpt hasTheSameValueAs bOpt` is equivalent to `aOpt.exists(a => bOpt.contains(a))`. I find the latter to be a bit clumsy

**Pull request merged**